includes/degree-functions.php

```diff
@@ -3,6 +3,378 @@
+						<?php endif; ?>
+					</dl>
+				</div>
+
+				<div class="col-lg-2 text-center align-self-center">
+
+					<img class="icon-calendar img-fluid" style="max-height: 4em;" role="img"
+						src="https://www.ucf.edu/online/files/2019/12/Calendar-Icon-light-gray.svg" alt="">
+					<div class="h1 mt-2 mb-0">16</div>
+					<div class="h6 text-muted text-uppercase">Weeks</div>
+
+					<img class="icon-clock img-fluid" style="max-height: 4em;" role="img"
+						src="https://www.ucf.edu/online/files/2019/12/Clock-Icon-light-gray.svg" alt="">
+					<div class="h1 mt-2 mb-0">44</div>
+					<div class="h6 text-muted text-uppercase">Credit Hours</div>
+
+				</div>
+
+				<div class="col-lg-4 bg-secondary text-uppercase border-bottom mb-md-3">
+					<?php echo get_degree_tuition_markup( $post_meta, 'modern' ); ?>
+				</div>
+
+				<div class="col-lg-3 text-center align-self-center">
+					<?php if ( $promo_image ) : ?>
+					<img src="<?php echo $promo_image; ?>" class="img-fluid p-4" role="img">
+					<?php endif; ?>
+				</div>
+
+			</div>
+		</div>
+	</div>
+
+	<?php
+	return ob_get_clean();
+
+ }
+
+
+
+ /**
+ * Returns HTML for the degree description section.
+  *
+  * @since 3.4.0
+  * @author RJ Bruneel
+  * @return string HTML markup
+  *
+  */
+  function get_degree_description_modern_layout() {
+	$degree_description_modern_layout = get_field( 'degree_description_modern_layout' );
+
+	if( empty( $degree_description_modern_layout ) ) return '';
+
+	ob_start();
+	?>
+	<div class="container py-lg-3">
+		<div class="row my-lg-3">
+			<?php echo $degree_description_modern_layout; ?>
+		</div>
+	</div>
+	<?php
+
+	return ob_get_clean();
+  }
+
+
+
+  /**
+   * Returns HTML for the application deadline section.
+   *
+   * @since 3.4.0
+   * @author RJ Bruneel
+   * @return string HTML markup
+   *
+   */
+   function get_degree_application_deadline_modern_layout() {
+	 $application_deadline_first = get_field( 'application_deadline_first' );
+	 $application_deadline_second = get_field( 'application_deadline_second' );
+
+	 if( empty( $application_deadline_first ) && empty( $application_deadline_second ) ) return '';
+ 
+	 ob_start();
+	 ?>
+	 <div class="bg-default">
+		<div class="container py-lg-3">
+			<div class="row my-lg-3">
+				<div class="col text-center text-uppercase">
+					<?php echo $application_deadline_first; ?><br>
+					<?php echo $application_deadline_second; ?>
+				</div>
+			</div>
+		</div>
+	</div>
+	 <?php
+ 
+	 return ob_get_clean();
+   }
+
+
+
+ /**
+  * Returns HTML for the course overview section.
+  *
+  * @since 3.4.0
+  * @author RJ Bruneel
+  * @return string HTML markup
+  *
+  */
+  function get_degree_course_overview_modern_layout() {
+	$course_overview = get_field( 'course_overview' );
+
+	if( empty( $course_overview ) ) return '';
+
+	ob_start();
+	?>
+	<div class="bg-faded">
+		<div class="container py-lg-3">
+			<div class="row my-lg-3">
+				<?php echo $course_overview; ?>
```

Comment:
Course overview needs to be contained in a `.col`., or the surrounding `.row` needs to be removed.